wordpress-mobile/WordPress-iOS
# Jetpack Section: Display a warning if server credentials aren't configured

Fixes #15924

### Description
- Updated the Restore Options VC to display a warning if server credentials aren't configured yet

### How to test

1. Create a Jetpack site
2. Add a Jetpack Security Daily plan
3. Do NOT enable restores (i.e. don't configure your server credentials)
4. Go to My Site > Jetpack > Backup > ellipsis > Restore
5. ✅ Notice that the "Restore to this point" button is disabled
6. ✅ Notice that the restore type switches are disabled and cannot be toggled
7. Tap on the warning "Enter your server credentials..."
8. ✅ Notice that the Jetpack Settings page opens up in a web vc

### Screenshot

Restore Options | Tap on warning to open Jetpack Settings
--- | ---
![Simulator Screen Shot - iPhone 12 Pro - 2021-02-22 at 18 34 17](https://user-images.githubusercontent.com/6711616/108690440-b8dedc00-753d-11eb-9ea5-d05d67d7aa00.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-02-22 at 17 56 49](https://user-images.githubusercontent.com/6711616/108690464-c005ea00-753d-11eb-89c5-14b40fb7fcd1.png)



### PR submission checklist:
